packages/shared/src/dto/step-schemas/control-preview-issue.type.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Is there a specific reason for not following the convention here?‏

Reply:
mistake